### DISCUSSION TITLE
Markor v2.8: No more "[TOC]" Table of Contents?

### DISCUSSION THREAD

**Discussion by gitgerardo** [gsantner/markor#1485] [Q&A] [answered]
## General information
* App version:2.8.2
* Android version: 11
* Device: oneplus 6
* I understand that this issue will be closed without further notice in case I didn't describe the problem including general information, or didn't [search](https://github.com/gsantner/markor/issues?q=) for similar issues & [requests](https://github.com/gsantner/markor/discussions?discussions_q=).

### Description

It is not anymore possible to use 
"[TOC]"  in order to show table oft content in a markdown file as it was possible in  all the previous versions up to 2.8.0. .

**Comment by gsantner** [comment, marked as answer]
Check settings, you can now enable TOC per heading level 1-6.  
Due new granular configuration options, your previous "[x] TOC" setting might have been reset.

**Comment by gitgerardo** [comment]
Ah,ok! 
Sorry,didn't see that.
Thank you!
Gert

04.11.2021 19:58:18 Gregor Santner ***@***.***>:
